File: Domain/Brand.cs
using System;
using System.Collections.Generic;
namespace Domain
{
    public class Brand
    { 
        public Guid Id { get; set; }
        public string Name { get; set; }
        public DateTime DateOfEstablishment { get; set; }
        public string LogoUrl { get; set; }
        public string LandOfOrigin { get; set; }
        public string CityOfOrigin { get; set; }
        public virtual ICollection<Motofy> Motofies { get; set; }
        
    }
}

File: client-app/src/app/models/brand.ts
export interface IBrand {
    id: string,
    name : string,
    dateOfEstablishment : string,
    logourl : string,
    landOfOrigin : string,
    cityOfOrigin : string,
}

// export interface IBrandFormValues extends Partial<IBrand>{

// }

// export class BrandFormValues implements IBrandFormValues {

//     id: string = '';
//     name : string = '';
//     key: string = this.name;
    
//     constructor(init?: BrandFormValues ) {
//         if (init)
//         Object.assign(this, init);
//     }
// }